lib/matplotlib/dates.py

```diff
@@ -1409,13 +1411,21 @@ def tick_values(self, vmin, vmax):
         ymin = self.base.le(vmin.year) * self.base.step
         ymax = self.base.ge(vmax.year) * self.base.step
 
-        ticks = [vmin.replace(year=ymin, **self.replaced)]
+        vmin = vmin.replace(year=ymin, **self.replaced)
+        try:
+            ticks = [vmin.astimezone(self.tz)]
+        except ValueError:
+            raise ValueError('naive datetime objects cannot be used '
```

Comment:
This may be a good place to use the `raise ... from e`  functionality of python3 (https://www.python.org/dev/peps/pep-3134/)

```python
try:
    ticks = [vmin.astimezone(self.tz)]
except ValueError as E:
    raise ValueError() from E
```

Reply:
OH, Cool, didn't know about that.  I'll do soon.